lib/UnoCore/Targets/Android/app/src/main/res/drawable/splash_background.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+ 
+ <item android:drawable="@color/splashColor" />
+ 
+ <item>
+	 <bitmap
+	 android:gravity="center"
+	 android:src="@drawable/splash_logo" />
+ </item>
```

Comment:
It would be nice to be consistent about indentation across different XML-files, 4 spaces per level, and indent attributes one level more than their parent element.